backend/middleware/authenticateToken.js

```diff
@@ -1,33 +1,27 @@
-const jwt = require("jsonwebtoken")
-
+const jwt = require("jsonwebtoken");
 
 const authenticateToken = (req, res, next) => {
-
-    // Get the token from the auth header (where it would be stored)
-    const authHeader = req.headers["authentication"];
-    const token = authHeader.split(' ')[1]
-
-    console.log(authHeader);
-
-    // Verify that there is a token
-    if(!token) {
-        // Bad request
-        // Code subject to change
-        res.sendStatus(401);
+  // Get the token from the auth header (where it would be stored)
+  const authHeader = req.headers["authentication"];
+  const token = authHeader.split(" ")[1];
+
+  // Verify that there is a token
+  if (!token) {
+    // Bad request
+    // Code subject to change
+    res.sendStatus(401);
+  }
+
+  jwt.verify(token, process.env.JWT_SECRET, (err, id) => {
+    if (err) {
+      // Status subject to change
+      res.sendStatus(403);
     }
 
-    jwt.verify(token, process.env.JWT_SECRET, (err, id) => {
-        
-        if(err) {
-            // Status subject to change
-            res.sendStatus(403);
-        }
-
-        req.id = id;
-
-        next();
-    })
+    req.id = id;
```

Comment:
Since `req.id` is being set, then for the controllers that are extracting the `id` from the token, we can remove those since the id is already set in `req.id`

Reply:
Fixed: Changes are reflected in recent commits